analytics-api-core/src/main/scala/org/ekstep/analytics/api/Model.scala

```diff
@@ -19,7 +19,9 @@ case class Request(filters: Option[Map[String, AnyRef]], config: Option[Map[Stri
 									 producer: Option[String] = None, tag: Option[String], dataset: Option[String], datasetConfig: Option[Map[String, Any]],
 									 requestedBy: Option[String], encryptionKey: Option[String], datasetType: Option[String], version: Option[String],
 									 visibility: Option[String], authorizedRoles: Option[List[String]], availableFrom: Option[String],
-									 sampleRequest: Option[String], sampleResponse: Option[String]);
+									 sampleRequest: Option[String], sampleResponse: Option[String], validationJson: Option[Map[String, Any]],
+									 druidQuery: Option[Map[String, Any]], limits: Option[Map[String, Any]], supportedFormats: Option[String],
```

Comment:
supportedFormats can be an array right? you can support both csv and json

Reply:
Updated supportedFormats type to array